libs/@local/harpc/tower/src/body/size_hint.rs

```diff
@@ -0,0 +1,86 @@
+//! Adapted and vendored from `http-body` crate (https://docs.rs/http-body/latest/src/http_body/size_hint.rs.html)
+
+/// A `Body` size hint
+///
+/// The default implementation returns:
+///
+/// * 0 for `lower`
+/// * `None` for `upper`.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Default)]
+pub struct SizeHint {
+    lower: u64,
+    upper: Option<u64>,
```

Comment:
Any specific reason why the fields are private with getters?

Reply:
this has been kept the same as in [http-body](https://docs.rs/http-body/latest/http_body/struct.SizeHint.html).